Week 5/MovieAppW5/app/src/main/java/com/janewaitara/movieapp/MovieAdapter.kt

```diff
@@ -0,0 +1,136 @@
+package com.janewaitara.movieapp
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class MovieAdapter(val clickListener: MovieListClickListener) : RecyclerView.Adapter<MovieAdapter.MovieViewHolder>() {
+
+    private var movieList = mutableListOf(
```

Comment:
You've done a nice job setting up your dao and database but unfortunately the recyclerview is still displaying this list and not storing it in the database.